src/OrbitQt/orbitmainwindow.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>2481</width>
-    <height>1138</height>
+    <width>1678</width>
+    <height>1016</height>
```

Comment:
Was this intended?

Reply:
No, this was simply because I saved the main window file. I've reverted the change.